### PR TITLE
Issue firefox#43 - style and correctness fixes for shared/tooltip.js

### PR DESCRIPTION
**Pull request by zombie**
related to privly/privly-firefox#43, based on notes from [Code Climate page for shared/tooltip.js](https://codeclimate.com/github/privly/privly-applications/shared/javascripts/tooltip.js)

### CODE REVIEW

File: shared/javascripts/tooltip.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /**
  * @fileOverview This script defines a tooltip to indicate that the content is
  * not a natural element of the page.
- * 
+ *
```
Comment (zombie) [inline shared/javascripts/tooltip.js]: All these whitespace removals on empty lines are done by my code editor automatically, as that's how most projects prefer to keep their files. If you disagree, I can amend the PR without them.

File: shared/javascripts/tooltip.js [hunk 2]
```diff
@@ -49,18 +49,18 @@ var privlyTooltip = {
...
-      privlyTooltip.tooltipMessage = privlyTooltip.appName
-        + ":\u00A0" // non-breaking space
-        + newMessage
-        + ", from "
-        + dataDomain;
+      privlyTooltip.tooltipMessage = privlyTooltip.appName +
+        ":\u00A0" +    // non-breaking space
+        newMessage +
+        ", from " +
+        dataDomain;
```
Comment (zombie) [inline shared/javascripts/tooltip.js]: This is also (mostly) a matter of style that is preferred by some projects in my experience, but if you disagree, I can again revert the change from this PR.

File: shared/javascripts/tooltip.js [hunk 3]
```diff
@@ -78,18 +78,18 @@ var privlyTooltip = {
      * for anything other than the glyph.
      */
     generateNewGlyph: function(){
-      
+
       var glyphString, glyphColor;
       glyphColor = Math.floor(Math.random()*16777215).toString(16);
       glyphString = ((Math.random() < 0.5) ? "false" : "true");
-      for(i = 0; i < 14; i++) {
+      for (var i = 0; i < 14; i++) {
```
Comment (zombie) [inline shared/javascripts/tooltip.js]: This was just plain wrong, polluting the global namespace.

File: shared/javascripts/tooltip.js [hunk 4]
```diff
@@ -191,10 +191,10 @@ var privlyTooltip = {
 
       var tbody = document.createElement("tbody");
 
-      for(i = 0; i < 5; i++) {
+      for (var i = 0; i < 5; i++) {
```
Comment (zombie) [inline shared/javascripts/tooltip.js]: ditto.

File: shared/javascripts/tooltip.js [hunk 5]
```diff
@@ -204,14 +204,14 @@ var privlyTooltip = {
           // Fill only the first three columns with the coresponding values from glyphArray[]
           // The rest of two columns are simetrical to the first two
           if(j <= 2) {
-            if(glyphArray[i * 3 + j] == "true") {
+            if(glyphArray[i * 3 + j] === "true") {
```
Comment (zombie) [inline shared/javascripts/tooltip.js]: This is a common practice to have a simple and blanket "always use strict equals comparisons", but again, I can leave it out of this PR if you prefer.

File: shared/javascripts/tooltip.js (hunk 2 above)
Comment (smcgregor) [inline shared/javascripts/tooltip.js]: Looks good.

File: shared/javascripts/tooltip.js (hunk 1 above)
Comment (smcgregor) [inline shared/javascripts/tooltip.js]: I generally like having whitespace cleanup commits separate, but it is no big deal in this case.

File: shared/javascripts/tooltip.js (hunk 3 above)
Comment (smcgregor) [inline shared/javascripts/tooltip.js]: Yup.

File: shared/javascripts/tooltip.js (hunk 5 above)
Comment (smcgregor) [inline shared/javascripts/tooltip.js]: Since this is a semantic change, I'll need to test it. However, this is good.

**Pull request merged**